lib/libpcp/src/pcp_event_handler.c

```diff
@@ -61,6 +61,7 @@
 
 #define MIN(a, b) (a<b?a:b)
 #define MAX(a, b) (a>b?a:b)
+// DISCUSSME: what should we do here?
```

Comment:
This is a 3rd-party library we've vendored into the FSO source tree and it appears to only be used to support port forwarding in the multiplayer code. None of this should be changed at all.

Reply:
Only @notimaginative would know

Reply:
Third party stuff should definitely *not* be changed if at all possible. It just makes it more difficult to merge in upstream changes later on.

And the PCP stuff is covered by multiple RFCs so rng usage here can be assumed to be following the spec.

Reply:
@notimaginative sure, makes sense. I read this comment after reading your "Do nothing" comment (reading these out of order) so I'll scrap the C compatibility layer.